src/redirects.js

```diff
@@ -141,3 +118,48 @@ export const setup = (app) => {
   }
 
 };
+
+
+/**
+ * Produces a list of dataset redirects. These are defined in a stand-alone
+ * function so that they can be easily reused by the resource indexer to
+ * associate "old" filenames with their new dataset path.
```

Comment:
_non-blocking_

Why a function and not just a static list?

Reply:
Hmm. I don't find those compelling in this case, FWIW.

- You can use a JSDoc comment with plain old variables.
- Checks can be done on a plain old variable too, done immediately after. (Or in the example of slashes handling like you give, the values can just be canonicalized with a `.map()` instead of erroring.)
- Deferred execution can be nice, but as you say is not applicable here at all.